source/api_documentation/transactions_api/_advertiser_param_table.rst

```diff
@@ -173,6 +173,14 @@
     - Call Start Time (XML formatted)
     - Start of the call in Soap XML formatted time.
 
+  * - start_time_network_timezone
+    - Call Start Time Network Timezone
+    - Start of the call in the networks's time zone, followed by offset from GMT.
```

Comment:
I do not believe it technically is because it uses a space instead of a `T` as the separator between the date and the time portion of the timestamp.